src/sqlfluff/cli/commands.py

```diff
@@ -514,6 +514,16 @@ def dump_file_payload(filename: Optional[str], payload: str):
     cls=DeprecatedOption,
     deprecated=["--disable_progress_bar"],
 )
+@click.option(
```

Comment:
Should we consider adding this to `fix` and `parse` as well?

Reply:
I considered this 🤔 - it's a good question.

For `parse` I thought that we get that information from `lint` already so it felt like duplication. Arguably some rules might behave differently during `fix` than during `lint` - so there's a good argument for `fix`.

On balance though, I thought for the purpose we need it for now, we get the bulk of the time from the parsing and linting steps, which are covered nicely by `lint`. Yes we get slightly different information from `fix` but not drastically.

So, my roundabout answer I think is that I'm tempted not to for now. There will almost certainly be another iteration of this (hence the `beta` warning), where we might want some more metadata. At that stage I think we re-evaluate this question again.

Reply:
No worries, very nitpicky question.